test/gas.js

```diff
@@ -19,6 +19,159 @@ process.removeAllListeners("uncaughtException");
 
 let mnemonic = "candy maple cake sugar pudding cream honey rich smooth crumble sweet treat";
 
+describe("EIP150 Gas Estimation: ", async function() {
+  const bootstrap = require("./helpers/bootstrap");
+  // const location = path.join(__dirname, "contracts", `${contractName}.sol`);
+  const ContractFactory = bootstrap("ContractFactory");
+  const TestDepth = bootstrap("TestDepth");
+  const Donation = bootstrap("Donation", ["Fib"]);
+  const Fib = bootstrap("Fib");
+  it("Should estimate gas perfectly with EIP150 - recursive CALL", async() => {
+    const { accounts, instance } = Fib;
+    const index = 5;
+    const est = await instance.methods.calc(index).estimateGas();
+    try {
+      await instance.methods.calc(index).send({
+        from: accounts[0],
+        gas: est - 1
+      });
+      assert.fail("Passed? SANITY CHECK");
+    } catch (e) {
+      assert("FAILED: not enough gas (expected)");
```

Comment:
Can you add the failing gas estimate to the failure message everywhere so we can get can take that value and test with it manually if we do get a failure?